hack/make/build-deb

```diff
@@ -58,9 +69,28 @@ set -e
 			FROM $image
 			WORKDIR /usr/src/docker
 			COPY . /usr/src/docker
-			RUN mkdir -p /go/src/github.com/docker \
+			RUN mkdir -p /go/src/github.com/docker && mkdir -p /go/src/github.com/opencontainers \
 				&& ln -snf /usr/src/docker /go/src/github.com/docker/docker
```

Comment:
@jfrazelle do you know if there's a reason why we don't simply `COPY` directly to `/go/src/github.com/docker/docker` instead of `/usr/src/docker` and then symlinking?